CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Printing the deposit address to the terminal as a QR code.
+  Users who are doing automated scripts or integrating with other software can use the `--json` flag to avoid this behaviour. It won't print a QR code if the `--json` flag is being passed to the application.
```

Comment:
From the way you worded this, I think there was a bit of a misunderstanding here. The `--json` flag doesn't exist to avoid the QRCode printing. The `--json` flag toggles whether or not the logs are printed in JSON format. It will make more sense once you try it!

Hence, what we should mention in the changelog is the fact that `--json` is conceptually incompatible with printing a QRCode and hence will be skipped.

Something like:

```suggestion
  To not break automated scripts or integrations with other software, this behaviour is disabled if `--json` is passed to the application.
```

Reply:
Yeah I tried the `--json` flag to test it, its prints the logs in JSON format. This was a bad wording choice by me.